code/modules/unit_tests/organs.dm

```diff
@@ -0,0 +1,174 @@
+/**
+ * Organ Unit Tests
+ *
+ * Test 1. organ_sanity:
```

Comment:
This entire comment shouldn't exist because now we need to update two places at once. Keep the documentation to the individual tests.